packages/frontend/src/components/dashboard/TemplateTable/Container.vue

```diff
@@ -122,73 +121,101 @@ const isAnalyzing = computed(() => {
 const defaultPorts = [80, 443];
 
 const getURLColumnValue = (template: TemplateDTO) => {
-  let url = `${template.meta.method} ${template.meta.isTls ? 'https' : 'http'}://${ template.meta.host }`;
+  let url = `${template.meta.method} ${
+    template.meta.isTls ? "https" : "http"
+  }://${template.meta.host}`;
   if (
-    !defaultPorts.includes(template.meta.port) || 
+    !defaultPorts.includes(template.meta.port) ||
     (template.meta.port === 443 && !template.meta.isTls) ||
     (template.meta.port === 80 && template.meta.isTls)
   ) {
     url += `:${template.meta.port}`;
   }
   url += template.meta.path;
-  if (url.length > 130) {
-    url = url.substring(0, 126) + "...";
+  if (url.length > 96) {
+    url = url.substring(0, 96) + "...";
   }
   return url;
-}
+};
 
 const onTemplateUpdate = (
   template: TemplateDTO,
   field: string,
-  newValue: unknown,
+  newValue: unknown
 ) => {
   service.updateTemplate(template.id, { ...template, [field]: newValue });
 };
-</script>
 
+const isSmallScreen = useMediaQuery("(max-width: 1150px)");
```

Comment:
Do you think we can leverage tailwind instead of doing responsiveness via JS?

https://tailwindcss.com/docs/responsive-design

We can do things like `class="lg:flex-col lg:gap-4"` to make styling be applied only to large screens, etc...